sources/api/migration/migrations/v1.15.0/oci-defaults-resource-setting/src/main.rs

```diff
@@ -0,0 +1,43 @@
+use migration_helpers::common_migrations::{AddPrefixesMigration, NoOpMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new resource limit settings for configuring the default OCI runtime spec
+/// like max-address-space, max-core-file-size, max-cpu-time, max-data-size, max-file-locks
+/// max-file-size, max-locked-memory, max-msgqueue-size, max-nice-priority, max-pending-signals,
+/// max-processes, max-realtime-priority, max-realtime-timeout, max-resident-set, max-stack-size.
```

Comment:
nit: no need to list all of these prefixes again:
```suggestion
/// We added new resource limit settings for configuring the default OCI runtime spec.
```